airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -143,7 +143,10 @@ def glue_job_hook(self) -> GlueJobHook:
             s3_hook = S3Hook(aws_conn_id=self.aws_conn_id)
             script_name = os.path.basename(self.script_location)
             s3_hook.load_file(
-                self.script_location, self.s3_artifacts_prefix + script_name, bucket_name=self.s3_bucket
+                self.script_location,
+                self.s3_artifacts_prefix + script_name,
+                bucket_name=self.s3_bucket,
+                replace=True,
```

Comment:
Are you sure that this is right for ALL usages of the operator?
If not then this parameter should be set on the operator level thus users can decide on the hebavior they want

Reply:
Good point. I do not know that this is the right approach for all potential use cases and I agree that implementing this as a parameter in the operator level is the defensive approach. I've just commited an updated implementation.